src/main/kotlin/org/jetbrains/research/ml/codetracker/ui/Controller.kt

```diff
@@ -245,14 +264,15 @@ class Controller(val project: Project, val scale: Double) {
     }
 
     fun setTaskInfo(task: Task) {
-        taskNameText.text = task.name
-        taskDescriptionText.text = task.description
-        taskInputText.text = task.input
-        taskOutputText.text = task.output
-
-        setExample(task.example_1, firstExampleInput, firstExampleOutput)
-        setExample(task.example_2, secondExampleInput, secondExampleOutput)
-        setExample(task.example_3, thirdExampleInput, thirdExampleOutput)
+        // Todo: get real info
+        taskNameText.text = "pies"
+        taskDescriptionText.text = "pies"
+        taskInputText.text = "pies"
+        taskOutputText.text = "pies"
```

Comment:
Do we still need unreal info here? I thought it's possible to get tasks from the server now